2024-08-18 00:56:20,547 INFO    StreamThr :9024 [internal.py:wandb_internal():85] W&B internal server running at pid: 9024, started at: 2024-08-18 00:56:20.544672
2024-08-18 00:56:20,549 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: status
2024-08-18 00:56:20,550 INFO    WriterThread:9024 [datastore.py:open_for_write():87] open: /Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/wandb/run-20240818_005613-kkoza7j3/run-kkoza7j3.wandb
2024-08-18 00:56:20,552 DEBUG   SenderThread:9024 [sender.py:send():391] send: header
2024-08-18 00:56:20,552 DEBUG   SenderThread:9024 [sender.py:send():391] send: run
2024-08-18 00:56:21,362 INFO    SenderThread:9024 [dir_watcher.py:__init__():211] watching files in: /Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/wandb/run-20240818_005613-kkoza7j3/files
2024-08-18 00:56:21,363 INFO    SenderThread:9024 [sender.py:_start_run_threads():1200] run started: kkoza7j3 with start time 1723922780.130782
2024-08-18 00:56:21,384 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: check_version
2024-08-18 00:56:21,385 DEBUG   SenderThread:9024 [sender.py:send_request():418] send_request: check_version
2024-08-18 00:56:21,564 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: run_start
2024-08-18 00:56:21,567 DEBUG   HandlerThread:9024 [system_info.py:__init__():26] System info init
2024-08-18 00:56:21,567 DEBUG   HandlerThread:9024 [system_info.py:__init__():41] System info init done
2024-08-18 00:56:21,567 INFO    HandlerThread:9024 [system_monitor.py:start():194] Starting system monitor
2024-08-18 00:56:21,567 INFO    SystemMonitor:9024 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-08-18 00:56:21,567 INFO    HandlerThread:9024 [system_monitor.py:probe():214] Collecting system info
2024-08-18 00:56:21,569 INFO    SystemMonitor:9024 [interfaces.py:start():188] Started cpu monitoring
2024-08-18 00:56:21,571 INFO    SystemMonitor:9024 [interfaces.py:start():188] Started disk monitoring
2024-08-18 00:56:21,581 INFO    SystemMonitor:9024 [interfaces.py:start():188] Started gpuapple monitoring
2024-08-18 00:56:21,592 INFO    SystemMonitor:9024 [interfaces.py:start():188] Started memory monitoring
2024-08-18 00:56:21,594 INFO    SystemMonitor:9024 [interfaces.py:start():188] Started network monitoring
2024-08-18 00:56:21,964 ERROR   gpuapple  :9024 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-18 00:56:22,295 ERROR   HandlerThread:9024 [gpu_apple.py:probe():176] GPU stats error: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 164, in probe
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-18 00:56:22,296 DEBUG   HandlerThread:9024 [system_info.py:probe():152] Probing system
2024-08-18 00:56:22,301 DEBUG   HandlerThread:9024 [system_info.py:_probe_git():137] Probing git
2024-08-18 00:56:22,336 DEBUG   HandlerThread:9024 [system_info.py:_probe_git():145] Probing git done
2024-08-18 00:56:22,337 DEBUG   HandlerThread:9024 [system_info.py:probe():200] Probing system done
2024-08-18 00:56:22,337 DEBUG   HandlerThread:9024 [system_monitor.py:probe():223] {'os': 'macOS-12.6.3-arm64-arm-64bit', 'python': '3.10.2', 'heartbeatAt': '2024-08-17T19:26:22.297066', 'startedAt': '2024-08-17T19:26:13.816886', 'docker': None, 'cuda': None, 'args': (), 'state': 'running', 'program': '<python with no main file>', 'codePathLocal': None, 'git': {'remote': 'https://github.com/Dhanunjaikumar369/RAG.git', 'commit': '684424abfd9ac4bd0c51164c465bdf977cca0a4d'}, 'email': 'Dhanunjai.kumar369@gmail.com', 'root': '/Users/dhanunjaikumar/Downloads/git_test/RAG', 'host': 'Dhanunjais-MacBook-Air.local', 'username': 'dhanunjaikumar', 'executable': '/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/bin/python', 'cpu_count': 8, 'cpu_count_logical': 8, 'cpu_freq': {'current': 3204, 'min': 600, 'max': 3204}, 'cpu_freq_per_core': [{'current': 3204, 'min': 600, 'max': 3204}], 'disk': {'/': {'total': 228.27386474609375, 'used': 14.360218048095703}}, 'gpuapple': {'type': 'arm', 'vendor': 'Apple'}, 'memory': {'total': 8.0}}
2024-08-18 00:56:22,337 INFO    HandlerThread:9024 [system_monitor.py:probe():224] Finished collecting system info
2024-08-18 00:56:22,337 INFO    HandlerThread:9024 [system_monitor.py:probe():227] Publishing system info
2024-08-18 00:56:22,339 INFO    HandlerThread:9024 [system_monitor.py:probe():229] Finished publishing system info
2024-08-18 00:56:22,341 DEBUG   SenderThread:9024 [sender.py:send():391] send: files
2024-08-18 00:56:22,341 INFO    SenderThread:9024 [sender.py:_save_file():1466] saving file wandb-metadata.json with policy now
2024-08-18 00:56:22,372 INFO    Thread-24 :9024 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/wandb/run-20240818_005613-kkoza7j3/files/wandb-metadata.json
2024-08-18 00:56:22,973 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: python_packages
2024-08-18 00:56:22,974 DEBUG   SenderThread:9024 [sender.py:send_request():418] send_request: python_packages
2024-08-18 00:56:22,977 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:56:22,977 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: stop_status
2024-08-18 00:56:22,979 DEBUG   SenderThread:9024 [sender.py:send_request():418] send_request: stop_status
2024-08-18 00:56:23,382 INFO    Thread-24 :9024 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/wandb/run-20240818_005613-kkoza7j3/files/requirements.txt
2024-08-18 00:56:23,425 DEBUG   SenderThread:9024 [sender.py:send():391] send: telemetry
2024-08-18 00:56:23,983 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:56:24,308 ERROR   gpuapple  :9024 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-18 00:56:24,379 INFO    Thread-24 :9024 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/wandb/run-20240818_005613-kkoza7j3/files/output.log
2024-08-18 00:56:24,982 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:56:25,257 INFO    wandb-upload_0:9024 [upload_job.py:push():130] Uploaded file /var/folders/sg/4hfsbh3s2p90p3wnmxhqpb8c0000gn/T/tmp_5i_gz58wandb/g162507g-wandb-metadata.json
2024-08-18 00:56:25,986 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:56:25,995 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: status_report
2024-08-18 00:56:26,388 INFO    Thread-24 :9024 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/wandb/run-20240818_005613-kkoza7j3/files/output.log
2024-08-18 00:56:26,937 ERROR   gpuapple  :9024 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-18 00:56:26,995 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:56:27,994 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:56:28,998 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:56:29,233 ERROR   gpuapple  :9024 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-18 00:56:29,999 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:56:31,002 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:56:31,584 ERROR   gpuapple  :9024 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-18 00:56:31,991 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: status_report
2024-08-18 00:56:32,002 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:56:33,006 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:56:33,896 ERROR   gpuapple  :9024 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-18 00:56:34,009 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:56:35,012 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:56:36,016 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:56:36,111 ERROR   gpuapple  :9024 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-18 00:56:37,005 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: status_report
2024-08-18 00:56:37,021 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:56:37,979 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: stop_status
2024-08-18 00:56:37,980 DEBUG   SenderThread:9024 [sender.py:send_request():418] send_request: stop_status
2024-08-18 00:56:38,025 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:56:38,305 ERROR   gpuapple  :9024 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-18 00:56:39,029 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:56:40,040 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:56:40,531 ERROR   gpuapple  :9024 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-18 00:56:41,037 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:56:42,038 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:56:42,401 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: status_report
2024-08-18 00:56:42,750 ERROR   gpuapple  :9024 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-18 00:56:43,041 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:56:44,045 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:56:45,049 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:56:45,051 ERROR   gpuapple  :9024 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-18 00:56:46,052 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:56:47,062 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:56:47,386 ERROR   gpuapple  :9024 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-18 00:56:47,999 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: status_report
2024-08-18 00:56:48,062 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:56:49,064 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:56:49,584 ERROR   gpuapple  :9024 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-18 00:56:50,068 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:56:51,073 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:56:51,958 ERROR   gpuapple  :9024 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-18 00:56:52,078 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:56:52,984 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: stop_status
2024-08-18 00:56:52,985 DEBUG   SenderThread:9024 [sender.py:send_request():418] send_request: stop_status
2024-08-18 00:56:53,080 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:56:53,353 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: status_report
2024-08-18 00:56:54,083 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:56:54,277 ERROR   gpuapple  :9024 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-18 00:56:54,504 INFO    Thread-24 :9024 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/wandb/run-20240818_005613-kkoza7j3/files/config.yaml
2024-08-18 00:56:55,109 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:56:56,096 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:56:56,653 ERROR   gpuapple  :9024 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-18 00:56:57,099 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:56:58,104 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:56:58,942 ERROR   gpuapple  :9024 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-18 00:56:59,019 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: status_report
2024-08-18 00:56:59,109 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:00,113 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:01,121 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:01,468 ERROR   gpuapple  :9024 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-18 00:57:02,119 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:03,124 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:03,809 ERROR   gpuapple  :9024 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-18 00:57:04,124 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:05,014 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: status_report
2024-08-18 00:57:05,131 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:06,137 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:06,175 ERROR   gpuapple  :9024 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-18 00:57:07,139 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:07,987 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: stop_status
2024-08-18 00:57:07,988 DEBUG   SenderThread:9024 [sender.py:send_request():418] send_request: stop_status
2024-08-18 00:57:08,142 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:08,321 ERROR   gpuapple  :9024 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-18 00:57:09,143 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:10,149 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:10,424 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: status_report
2024-08-18 00:57:10,677 ERROR   gpuapple  :9024 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-18 00:57:11,147 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:12,149 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:13,026 ERROR   gpuapple  :9024 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-18 00:57:13,151 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:14,154 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:15,167 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:15,360 ERROR   gpuapple  :9024 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-18 00:57:16,038 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: status_report
2024-08-18 00:57:16,161 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:17,164 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:17,709 ERROR   gpuapple  :9024 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-18 00:57:18,165 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:19,171 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:19,851 ERROR   gpuapple  :9024 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-18 00:57:20,175 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:21,180 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:21,598 DEBUG   SystemMonitor:9024 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-08-18 00:57:21,600 DEBUG   SenderThread:9024 [sender.py:send():391] send: telemetry
2024-08-18 00:57:21,601 DEBUG   SenderThread:9024 [sender.py:send():391] send: stats
2024-08-18 00:57:21,601 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: status_report
2024-08-18 00:57:22,098 ERROR   gpuapple  :9024 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-18 00:57:22,185 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:22,989 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: stop_status
2024-08-18 00:57:22,989 DEBUG   SenderThread:9024 [sender.py:send_request():418] send_request: stop_status
2024-08-18 00:57:23,189 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:24,193 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:24,247 ERROR   gpuapple  :9024 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-18 00:57:25,197 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:26,202 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:26,388 ERROR   gpuapple  :9024 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-18 00:57:27,207 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:27,347 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: status_report
2024-08-18 00:57:28,210 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:28,626 ERROR   gpuapple  :9024 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-18 00:57:28,671 INFO    Thread-24 :9024 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/wandb/run-20240818_005613-kkoza7j3/files/config.yaml
2024-08-18 00:57:29,214 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:30,213 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:30,805 ERROR   gpuapple  :9024 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-18 00:57:31,216 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:32,218 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:32,996 ERROR   gpuapple  :9024 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-18 00:57:33,031 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: status_report
2024-08-18 00:57:33,223 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:34,228 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:35,242 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:35,364 ERROR   gpuapple  :9024 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-18 00:57:36,431 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:37,436 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:37,636 ERROR   gpuapple  :9024 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 420, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/.venv/lib/python3.10/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-18 00:57:37,997 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: stop_status
2024-08-18 00:57:37,997 DEBUG   SenderThread:9024 [sender.py:send_request():418] send_request: stop_status
2024-08-18 00:57:38,420 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: status_report
2024-08-18 00:57:38,467 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:38,748 DEBUG   SenderThread:9024 [sender.py:send():391] send: telemetry
2024-08-18 00:57:38,749 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: server_info
2024-08-18 00:57:38,750 DEBUG   SenderThread:9024 [sender.py:send_request():418] send_request: server_info
2024-08-18 00:57:39,078 DEBUG   SenderThread:9024 [sender.py:send():391] send: exit
2024-08-18 00:57:39,078 INFO    SenderThread:9024 [sender.py:send_exit():598] handling exit code: 0
2024-08-18 00:57:39,078 INFO    SenderThread:9024 [sender.py:send_exit():600] handling runtime: 77
2024-08-18 00:57:39,081 INFO    SenderThread:9024 [sender.py:_save_file():1466] saving file wandb-summary.json with policy end
2024-08-18 00:57:39,082 INFO    SenderThread:9024 [sender.py:send_exit():606] send defer
2024-08-18 00:57:39,082 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: defer
2024-08-18 00:57:39,082 INFO    HandlerThread:9024 [handler.py:handle_request_defer():184] handle defer: 0
2024-08-18 00:57:39,083 DEBUG   SenderThread:9024 [sender.py:send_request():418] send_request: defer
2024-08-18 00:57:39,083 INFO    SenderThread:9024 [sender.py:send_request_defer():622] handle sender defer: 0
2024-08-18 00:57:39,083 INFO    SenderThread:9024 [sender.py:transition_state():626] send defer: 1
2024-08-18 00:57:39,084 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: defer
2024-08-18 00:57:39,084 INFO    HandlerThread:9024 [handler.py:handle_request_defer():184] handle defer: 1
2024-08-18 00:57:39,085 DEBUG   SenderThread:9024 [sender.py:send_request():418] send_request: defer
2024-08-18 00:57:39,085 INFO    SenderThread:9024 [sender.py:send_request_defer():622] handle sender defer: 1
2024-08-18 00:57:39,085 INFO    SenderThread:9024 [sender.py:transition_state():626] send defer: 2
2024-08-18 00:57:39,085 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: defer
2024-08-18 00:57:39,085 INFO    HandlerThread:9024 [handler.py:handle_request_defer():184] handle defer: 2
2024-08-18 00:57:39,085 INFO    HandlerThread:9024 [system_monitor.py:finish():203] Stopping system monitor
2024-08-18 00:57:39,086 DEBUG   SystemMonitor:9024 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-08-18 00:57:39,086 INFO    HandlerThread:9024 [interfaces.py:finish():200] Joined cpu monitor
2024-08-18 00:57:39,086 DEBUG   SystemMonitor:9024 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-08-18 00:57:39,086 INFO    HandlerThread:9024 [interfaces.py:finish():200] Joined disk monitor
2024-08-18 00:57:39,087 INFO    HandlerThread:9024 [interfaces.py:finish():200] Joined gpuapple monitor
2024-08-18 00:57:39,087 INFO    HandlerThread:9024 [interfaces.py:finish():200] Joined memory monitor
2024-08-18 00:57:39,088 INFO    HandlerThread:9024 [interfaces.py:finish():200] Joined network monitor
2024-08-18 00:57:39,089 DEBUG   SenderThread:9024 [sender.py:send_request():418] send_request: defer
2024-08-18 00:57:39,089 INFO    SenderThread:9024 [sender.py:send_request_defer():622] handle sender defer: 2
2024-08-18 00:57:39,089 INFO    SenderThread:9024 [sender.py:transition_state():626] send defer: 3
2024-08-18 00:57:39,089 DEBUG   SenderThread:9024 [sender.py:send():391] send: stats
2024-08-18 00:57:39,090 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: defer
2024-08-18 00:57:39,090 INFO    HandlerThread:9024 [handler.py:handle_request_defer():184] handle defer: 3
2024-08-18 00:57:39,091 DEBUG   SenderThread:9024 [sender.py:send_request():418] send_request: defer
2024-08-18 00:57:39,091 INFO    SenderThread:9024 [sender.py:send_request_defer():622] handle sender defer: 3
2024-08-18 00:57:39,091 INFO    SenderThread:9024 [sender.py:transition_state():626] send defer: 4
2024-08-18 00:57:39,092 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: defer
2024-08-18 00:57:39,092 INFO    HandlerThread:9024 [handler.py:handle_request_defer():184] handle defer: 4
2024-08-18 00:57:39,093 DEBUG   SenderThread:9024 [sender.py:send_request():418] send_request: defer
2024-08-18 00:57:39,093 INFO    SenderThread:9024 [sender.py:send_request_defer():622] handle sender defer: 4
2024-08-18 00:57:39,093 INFO    SenderThread:9024 [sender.py:transition_state():626] send defer: 5
2024-08-18 00:57:39,094 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: defer
2024-08-18 00:57:39,095 INFO    HandlerThread:9024 [handler.py:handle_request_defer():184] handle defer: 5
2024-08-18 00:57:39,095 DEBUG   SenderThread:9024 [sender.py:send():391] send: summary
2024-08-18 00:57:39,096 INFO    SenderThread:9024 [sender.py:_save_file():1466] saving file wandb-summary.json with policy end
2024-08-18 00:57:39,097 DEBUG   SenderThread:9024 [sender.py:send_request():418] send_request: defer
2024-08-18 00:57:39,097 INFO    SenderThread:9024 [sender.py:send_request_defer():622] handle sender defer: 5
2024-08-18 00:57:39,097 INFO    SenderThread:9024 [sender.py:transition_state():626] send defer: 6
2024-08-18 00:57:39,097 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: defer
2024-08-18 00:57:39,097 INFO    HandlerThread:9024 [handler.py:handle_request_defer():184] handle defer: 6
2024-08-18 00:57:39,097 DEBUG   SenderThread:9024 [sender.py:send_request():418] send_request: defer
2024-08-18 00:57:39,097 INFO    SenderThread:9024 [sender.py:send_request_defer():622] handle sender defer: 6
2024-08-18 00:57:39,106 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: status_report
2024-08-18 00:57:39,463 INFO    SenderThread:9024 [sender.py:transition_state():626] send defer: 7
2024-08-18 00:57:39,463 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: defer
2024-08-18 00:57:39,463 INFO    HandlerThread:9024 [handler.py:handle_request_defer():184] handle defer: 7
2024-08-18 00:57:39,463 DEBUG   SenderThread:9024 [sender.py:send_request():418] send_request: defer
2024-08-18 00:57:39,463 INFO    SenderThread:9024 [sender.py:send_request_defer():622] handle sender defer: 7
2024-08-18 00:57:39,731 INFO    Thread-24 :9024 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/wandb/run-20240818_005613-kkoza7j3/files/config.yaml
2024-08-18 00:57:39,732 INFO    Thread-24 :9024 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/wandb/run-20240818_005613-kkoza7j3/files/output.log
2024-08-18 00:57:39,732 INFO    Thread-24 :9024 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/wandb/run-20240818_005613-kkoza7j3/files/wandb-summary.json
2024-08-18 00:57:40,079 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: poll_exit
2024-08-18 00:57:40,732 INFO    Thread-24 :9024 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/wandb/run-20240818_005613-kkoza7j3/files/output.log
2024-08-18 00:57:41,612 INFO    SenderThread:9024 [sender.py:transition_state():626] send defer: 8
2024-08-18 00:57:41,612 DEBUG   SenderThread:9024 [sender.py:send_request():418] send_request: poll_exit
2024-08-18 00:57:41,612 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: defer
2024-08-18 00:57:41,613 INFO    HandlerThread:9024 [handler.py:handle_request_defer():184] handle defer: 8
2024-08-18 00:57:41,613 DEBUG   SenderThread:9024 [sender.py:send_request():418] send_request: defer
2024-08-18 00:57:41,613 INFO    SenderThread:9024 [sender.py:send_request_defer():622] handle sender defer: 8
2024-08-18 00:57:41,613 INFO    SenderThread:9024 [job_builder.py:build():443] Attempting to build job artifact
2024-08-18 00:57:41,614 INFO    SenderThread:9024 [job_builder.py:_get_source_type():572] is repo sourced job
2024-08-18 00:57:41,614 INFO    SenderThread:9024 [job_builder.py:_get_program_relpath():590] run is notebook based run
2024-08-18 00:57:41,614 WARNING SenderThread:9024 [job_builder.py:_log_if_verbose():274] Source type is set to 'repo' but some required information is missing from the environment. A job will not be created from this run. See https://docs.wandb.ai/guides/launch/create-job
2024-08-18 00:57:41,614 INFO    SenderThread:9024 [sender.py:transition_state():626] send defer: 9
2024-08-18 00:57:41,614 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: defer
2024-08-18 00:57:41,614 INFO    HandlerThread:9024 [handler.py:handle_request_defer():184] handle defer: 9
2024-08-18 00:57:41,614 DEBUG   SenderThread:9024 [sender.py:send_request():418] send_request: defer
2024-08-18 00:57:41,614 INFO    SenderThread:9024 [sender.py:send_request_defer():622] handle sender defer: 9
2024-08-18 00:57:41,614 INFO    SenderThread:9024 [dir_watcher.py:finish():358] shutting down directory watcher
2024-08-18 00:57:41,737 INFO    SenderThread:9024 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/wandb/run-20240818_005613-kkoza7j3/files/output.log
2024-08-18 00:57:41,738 INFO    SenderThread:9024 [dir_watcher.py:finish():388] scan: /Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/wandb/run-20240818_005613-kkoza7j3/files
2024-08-18 00:57:41,738 INFO    SenderThread:9024 [dir_watcher.py:finish():402] scan save: /Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/wandb/run-20240818_005613-kkoza7j3/files/requirements.txt requirements.txt
2024-08-18 00:57:41,739 INFO    SenderThread:9024 [dir_watcher.py:finish():402] scan save: /Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/wandb/run-20240818_005613-kkoza7j3/files/output.log output.log
2024-08-18 00:57:41,739 INFO    SenderThread:9024 [dir_watcher.py:finish():402] scan save: /Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/wandb/run-20240818_005613-kkoza7j3/files/config.yaml config.yaml
2024-08-18 00:57:41,741 INFO    SenderThread:9024 [dir_watcher.py:finish():402] scan save: /Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/wandb/run-20240818_005613-kkoza7j3/files/wandb-summary.json wandb-summary.json
2024-08-18 00:57:41,743 INFO    SenderThread:9024 [dir_watcher.py:finish():402] scan save: /Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/wandb/run-20240818_005613-kkoza7j3/files/wandb-metadata.json wandb-metadata.json
2024-08-18 00:57:41,743 INFO    SenderThread:9024 [sender.py:transition_state():626] send defer: 10
2024-08-18 00:57:41,743 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: defer
2024-08-18 00:57:41,745 INFO    HandlerThread:9024 [handler.py:handle_request_defer():184] handle defer: 10
2024-08-18 00:57:41,749 DEBUG   SenderThread:9024 [sender.py:send_request():418] send_request: defer
2024-08-18 00:57:41,749 INFO    SenderThread:9024 [sender.py:send_request_defer():622] handle sender defer: 10
2024-08-18 00:57:41,749 INFO    SenderThread:9024 [file_pusher.py:finish():169] shutting down file pusher
2024-08-18 00:57:42,086 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: poll_exit
2024-08-18 00:57:42,086 DEBUG   SenderThread:9024 [sender.py:send_request():418] send_request: poll_exit
2024-08-18 00:57:42,538 INFO    wandb-upload_0:9024 [upload_job.py:push():130] Uploaded file /Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/wandb/run-20240818_005613-kkoza7j3/files/requirements.txt
2024-08-18 00:57:42,737 INFO    wandb-upload_1:9024 [upload_job.py:push():130] Uploaded file /Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/wandb/run-20240818_005613-kkoza7j3/files/output.log
2024-08-18 00:57:42,769 INFO    wandb-upload_2:9024 [upload_job.py:push():130] Uploaded file /Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/wandb/run-20240818_005613-kkoza7j3/files/config.yaml
2024-08-18 00:57:42,778 INFO    wandb-upload_3:9024 [upload_job.py:push():130] Uploaded file /Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/wandb/run-20240818_005613-kkoza7j3/files/wandb-summary.json
2024-08-18 00:57:42,984 INFO    Thread-23 (_thread_body):9024 [sender.py:transition_state():626] send defer: 11
2024-08-18 00:57:42,984 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: defer
2024-08-18 00:57:42,984 INFO    HandlerThread:9024 [handler.py:handle_request_defer():184] handle defer: 11
2024-08-18 00:57:42,984 DEBUG   SenderThread:9024 [sender.py:send_request():418] send_request: defer
2024-08-18 00:57:42,984 INFO    SenderThread:9024 [sender.py:send_request_defer():622] handle sender defer: 11
2024-08-18 00:57:42,984 INFO    SenderThread:9024 [file_pusher.py:join():175] waiting for file pusher
2024-08-18 00:57:42,985 INFO    SenderThread:9024 [sender.py:transition_state():626] send defer: 12
2024-08-18 00:57:42,985 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: defer
2024-08-18 00:57:42,985 INFO    HandlerThread:9024 [handler.py:handle_request_defer():184] handle defer: 12
2024-08-18 00:57:42,985 DEBUG   SenderThread:9024 [sender.py:send_request():418] send_request: defer
2024-08-18 00:57:42,985 INFO    SenderThread:9024 [sender.py:send_request_defer():622] handle sender defer: 12
2024-08-18 00:57:42,985 INFO    SenderThread:9024 [file_stream.py:finish():601] file stream finish called
2024-08-18 00:57:43,092 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: poll_exit
2024-08-18 00:57:43,871 INFO    SenderThread:9024 [file_stream.py:finish():605] file stream finish is done
2024-08-18 00:57:43,872 INFO    SenderThread:9024 [sender.py:transition_state():626] send defer: 13
2024-08-18 00:57:43,872 DEBUG   SenderThread:9024 [sender.py:send_request():418] send_request: poll_exit
2024-08-18 00:57:43,872 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: defer
2024-08-18 00:57:43,873 INFO    HandlerThread:9024 [handler.py:handle_request_defer():184] handle defer: 13
2024-08-18 00:57:43,874 DEBUG   SenderThread:9024 [sender.py:send_request():418] send_request: defer
2024-08-18 00:57:43,874 INFO    SenderThread:9024 [sender.py:send_request_defer():622] handle sender defer: 13
2024-08-18 00:57:43,875 INFO    SenderThread:9024 [sender.py:transition_state():626] send defer: 14
2024-08-18 00:57:43,875 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: defer
2024-08-18 00:57:43,880 INFO    HandlerThread:9024 [handler.py:handle_request_defer():184] handle defer: 14
2024-08-18 00:57:43,880 DEBUG   SenderThread:9024 [sender.py:send():391] send: final
2024-08-18 00:57:43,881 DEBUG   SenderThread:9024 [sender.py:send():391] send: footer
2024-08-18 00:57:43,881 DEBUG   SenderThread:9024 [sender.py:send_request():418] send_request: defer
2024-08-18 00:57:43,881 INFO    SenderThread:9024 [sender.py:send_request_defer():622] handle sender defer: 14
2024-08-18 00:57:43,898 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: poll_exit
2024-08-18 00:57:43,899 DEBUG   SenderThread:9024 [sender.py:send_request():418] send_request: poll_exit
2024-08-18 00:57:43,914 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-18 00:57:43,929 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: get_summary
2024-08-18 00:57:43,930 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: sampled_history
2024-08-18 00:57:43,936 DEBUG   HandlerThread:9024 [handler.py:handle_request():158] handle_request: shutdown
2024-08-18 00:57:43,937 INFO    HandlerThread:9024 [handler.py:finish():880] shutting down handler
2024-08-18 00:57:44,903 INFO    SenderThread:9024 [sender.py:finish():1631] shutting down sender
2024-08-18 00:57:44,903 INFO    WriterThread:9024 [datastore.py:close():296] close: /Users/dhanunjaikumar/Downloads/git_test/RAG/Master-RAG-main/wandb/run-20240818_005613-kkoza7j3/run-kkoza7j3.wandb
2024-08-18 00:57:44,904 INFO    SenderThread:9024 [file_pusher.py:finish():169] shutting down file pusher
2024-08-18 00:57:44,904 INFO    SenderThread:9024 [file_pusher.py:join():175] waiting for file pusher
